presentations/2024-01-25_coffee-and-coding/index.qmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "Coffee and Coding"
+subtitle: "{targets}"
+author: "[Jacqueline Grout](mailto:jacqueline.grout1@nhs.net) and [Matt Dray](mailto:matt.dray@nhs.net)"
```

Comment:
I think this should just be your name; you've done all the work and I think you should present and demo the whole session.
```suggestion
author: "[Jacqueline Grout](mailto:jacqueline.grout1@nhs.net)"
```